featuretools/computational_backends/calculate_feature_matrix.py

```diff
@@ -230,14 +238,16 @@ def calculate_feature_matrix(features, entityset=None, cutoff_time=None, instanc
         cutoff_time_to_pass = cutoff_time
 
     chunk_size = _handle_chunk_size(chunk_size, cutoff_time.shape[0])
-
-    # make total 5% higher to allot time for wrapping up at end
-    progress_bar = make_tqdm_iterator(
-        total=cutoff_time.shape[0] / .95,
-        smoothing=.05,  # arbitrary selection close to 0, which would be no smoothing
-        bar_format=PBAR_FORMAT,
-        disable=(not verbose)
-    )
+    tqdm_options = {'total': (cutoff_time.shape[0] / .95),  # make total 5% higher to allot time for wrapping up at end
+                    'smoothing': .05,  # arbitrary selection close to 0, which would be no smoothing
+                    'bar_format': PBAR_FORMAT}
+    if progress_callback is not None:
```

Comment:
would it be cleaner if we set `disable: True` by default in `tqdm_options` above? Then the logic here could just be 
```
if verbose:
    tqdm_options.update({'disable': False})
elif progress_callback is not None:
    # allows us to utilize progress_bar updates without printing to anywhere
    tqdm_options.update({'file': open(os.devnull, 'w')})
```

Reply:
Updated! (I edited the suggestion slightly since above logic would disable `progress_bar` even if `progress_callback is not None`.